### DISCUSSION TITLE
Do we need to configure the targets.R code using dynamic branching?

### DISCUSSION THREAD

**Discussion by slevin75** [USGS-R/regional-hydrologic-forcings-ml#18] [General]
Help me think this through a little.  I set up the repository using static branching which assumes we have a set list of gages and that all those gages will make it through the tar_map block without failing in some way.  As I found out when I ran this for a larger portion of gages, this is problematic because if site has zero complete years, it will fail.   There are some sites, which, even though they are in gagesII don't actually have any daily dishcarge or don't have peak flow in NWIS, so they will fail at different places. I ran into a couple other odd situations as well, which I just manually removed for that period of record analysis.  

So what we need is a data screener that gets the data and then, if there isn't enough data or whatever, that site is dropped and everything else keeps going.  Is there a way to do that within the current set up or should this be reconfigured using dynamic branching? I'm starting to forget how dynamic branching works but I think it was able to accomodate that situation more easily than static branching.

**Comment by jds485** [comment]
As discussed in meeting, can use "Create issue from discussion" button (bottom right) to change to issue that moves the first few `tar_map` targets out of the map and writes a file with the gauges that we use in the new tar_map
